canonical_sphinx/theme/templates/components/edit-this-page.html

```diff
@@ -9,16 +9,16 @@
 </div>
 {%- endmacro -%}
 
-{%- if repo_folder -%}
-  {%- set docs_dir = repo_folder.strip("/") -%}
+{%- if repo_folder != "" -%}
+  {%- set docs_dir = "/" + repo_folder.strip("/") -%}
```

Comment:
```suggestion
  {%- set docs_dir = "/" + repo_folder.strip("/") -%}
  {%- if docs_dir == "/" -%}
    {%- set docs_dir = "" -%}
  {%- endif -%}
```

Not the prettiest, but this extra check should cover Anbox's case where `repo_folder = "/"`.

Reply:
That's a very good point. Nice catch.